test/setup_nodefs.js

```diff
@@ -1,3 +1,16 @@
+const statfsSync = require('fs').statfsSync;
+
+const DEFAULT_BLOCKS = 1e6;
+
+Module['preRun'] = () => {
+  try {
+    const statfs = statfsSync('/');
+    ENV.EXPECTED_BLOCKS = statfs.blocks.toString();
+  } catch (e) {
+    // Older versions of Node don't support statfsSync
+  }
+};
```

Comment:
If this is only needed for the one test I would just do this in `fstatfs.c` itself.  You can just do it in an EM_ASM or EM_JS block on startup?